Mathlib/Data/Bool/Count.lean

```diff
@@ -4,7 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Yury Kudryashov
 
 ! This file was ported from Lean 3 source module data.bool.count
-! leanprover-community/mathlib commit 2445c98ae4b87eabebdde552593519b9b6dc350c
+! leanprover-community/mathlib commit 8631e2d5ea77f6c13054d9151d82b83069680cb1
 ! Please do not edit these lines, except to modify the commit id
 ! if you have ported upstream changes.
 -/
```

Comment:
Please remove `Data.List.Count` from imports below to keep `import` lines in sync.

Reply:
Done in 75b9e3c98685a4778f81003af7d40d6e33e032e7